cloudhub/modules/ROOT/pages/_partials/ch-penetration-testing.adoc

```diff
@@ -0,0 +1,83 @@
+//Run Vulnerability Assessment and Penetration Tests
+
+//tag::intro[]
+If your company security policies require that you perform vulnerability and penetration testing against Anypoint Platform, you may commence testing without notifying Salesforce beforehand.
+
+All assessments must be performed in accordance with the https://help.salesforce.com/s/articleView?id=000392845&type=1[Security Assessment Agreement^].
+// end::intro[]
+
+//tag::runTest[]
+== Run Penetration and Vulnerability Tests
+
+Because penetration testing could interfere with other tenants, MuleSoft allows penetration testing on your workers but not on other Anypoint Platform services.
```

Comment:
```suggestion
Because penetration testing can interfere with other tenants, MuleSoft allows penetration testing on your workers but not on other Anypoint Platform services.
```